nexosim-util/src/helper_models.rs

```diff
@@ -0,0 +1,34 @@
+//! Helper models.
+//!
+//! This module contains helper models useful for simulation bench assembly.
+//!
+
+use std::time::Duration;
+
+use nexosim::model::{Context, InitializedModel, Model};
+
+/// A ticker model.
+///
+/// This model self-schedules with a provided period keeping simulation alive.
+pub struct Ticker {
+    /// Tick period in milliseconds.
+    tick: Duration,
+}
+
+impl Ticker {
+    /// Creates new `Ticker` with provided tick period in milliseconds.
```

Comment:
(same as before re: milliseconds)
I'd write:

Creates a new `Ticker` with the specified self-scheduling period.